docs/documentation/upgrading/topics/changes/changes-26_4_0.adoc

```diff
@@ -43,6 +43,28 @@ Custom caches can still be added without setting this option.
 When upgrading an existing deployment, remove all default cache configurations from your existing `conf/cache-ispn.xml`
 and use the `+--cache-...+` options to make changes for example to the cache sizes.
 
+=== RFC8414 compliant lookup of metadata
+
+{project_name} now exposes an RFC8414-compliant endpoint at the root URL level `/.well-known/` to allow clients to discover OAuth 2.0 Authorization Server Metadata by the issuer URL.
+
+Previously information was exposed via:
+
+[source]
+----
+https://keycloak.example.com/realms/{realm}/.well-known/oauth-authorization-server
+----
+
```

Comment:
Probably don't want to call this legacy, as that suggests it's being deprecated. Don't think we plan to do that anytime soon